docs/_sidebar.md

```diff
@@ -21,6 +21,7 @@ free to add sections and subsections to this sidebar.)
     - [Control](https://eclipse-edc.github.io/Connector/openapi/control-api/)
     - [DSP](https://eclipse-edc.github.io/Connector/openapi/dsp-api/)
     - [Public](https://eclipse-edc.github.io/Connector/openapi/public-api/)
+  - [Best Practices](../developer/best_practices.md)
```

Comment:
I don't think this should go under the `Connector` section, but stay in an overarching category because it's related to all the components